.travis.yml

```diff
@@ -29,9 +25,14 @@ jobs:
   - stage: pr previews
     script: echo "TODO"
   - stage: npm publish
-    script:
-    - "./utils/travis/getGit.sh"
-  - stage: storybook
+    deploy:
+      provider: npm
+      email: "$GITHUB_EMAIL"
+      api_key: "$NPM_TOKEN"
+      on:
+        branch: master
+        tags: true
```

Comment:
Looking over the docs, does this mean that `branch` is moot if we set `tags`? https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on

Also do we need `if` in the stages, if these are set?

Reply:
I guess we'll always need `if: branch = master AND type != pull_request` at least if `on.tags` invalidates `on.branch`